mappings/net/minecraft/util/math/ChunkPos.mapping

```diff
@@ -1,8 +1,16 @@
 CLASS net/minecraft/unmapped/C_ynrszrtu net/minecraft/util/math/ChunkPos
+	FIELD f_acuruuyv COORD_MASK J
```

Comment:
Is coord an abbreviation we commonly use for coordinate?

Reply:
Not a fan of it personally

Reply:
Coord seems to be the term mojang uses to refer to the conversion to long
![image](https://user-images.githubusercontent.com/47987888/156683546-bece93cd-4c6e-41ff-99c8-8781a335ff4c.png)
(0xFFFFFFFFL is `COORD_MASK`, and 32 is `COORD_BITS`)

Any suggestions?